test/tests/materials/get_material_property_names/get_material_property_any_block_id.i

```diff
@@ -54,7 +54,7 @@
 [UserObjects]
   [./get_material_block_names_test]
     type = GetMaterialPropertyBoundaryBlockNamesTest
-    expected_names = 'ANY_BLOCK_ID'
+    expected_names = 'top 0'
```

Comment:
Why is 'top' showing up here?

Reply:
`top` is a name assigned in the mesh generator. I did not check why `getMaterialPropertyBlockNames` can now return proper block names, which is a good thing.